src/parse/rules/c_rules.build_defs

```diff
@@ -27,8 +27,10 @@ def c_library(name:str, srcs:list=[], hdrs:list=[], private_hdrs:list=[], deps:l
       compiler_flags (list): Flags to pass to the compiler.
       linker_flags (list): Flags to pass to the linker; these will not be used here but will be
                            picked up by a c_binary or c_test rule.
-      pkg_config_libs (list): Libraries to declare a dependency on using pkg-config. Again, the ldflags
-                              will be picked up by c_binary or c_test rules.
+      pkg_config_libs (list): Libraries to declare a dependency on using `pkg-config --libs`. Again, the ldflags
```

Comment:
This isn't quite correct; it does also imply --cflags sometimes too.

Suggest leaving this one as it was before.